types.cc

```diff
@@ -360,6 +360,9 @@ struct string_type_impl : public concrete_type<sstring> {
     virtual bool is_byte_order_comparable() const override {
         return true;
     }
+    virtual bool is_string() const override {
```

Comment:
There is also an ASCII type.

Reply:
... of which this is a base class. :)

Reply:
I know someone who would have called this is_likeable()